### ISSUE TITLE
0.5 ETH is pretty steep price to create Kudos art, can we change this to a set dollar amount instead?

### ISSUE THREAD

**Issue by CryptoCutter** [gitcoinco/kudosbadges#341]
# Timeline

# Title

# Design Inspiration

# Down Payment

# Gen0 Owner Address

**Comment by Kamiel7** [comment]
Yes, that is a great idea, I actually,by mistake didn't specify that the price is for a finished price, also the price is not necessarily fixed, as there are many unknowns such as NFT market auction, and or privite listings.
You have given me an idea I must say, we may be able to get a few like minded individuals together and make and sell artworks , advertisment logos etc.

**Comment by yourmomma2000** [comment]
is that a steep price for the buyer or the seller? 🤗

**Comment by Kamiel7** [comment]
It's a sale price, make an offer, help a family in need, thanks and
blessings .
